src/App.css

```diff
@@ -28,44 +28,90 @@
   to { transform: rotate(360deg); }
 }
 
+.ambassadors {
+  display: flex;
+  flex-flow: row wrap;
+  /* width: 85%; */
+  height: 100px;
+  -webkit-margin-before: 1em;
+  -webkit-margin-after: 1em;
+  -webkit-margin-start: 0px;
+  -webkit-margin-end: 0px;
+  -webkit-padding-start: 40px
```

Comment:
you shouldn't need to do prefixing as create react app has this already built in. see https://github.com/facebook/create-react-app#whats-included

Also currently the way this code is writ non-webkit browsers wont have margin or padding as you have described - is that intentional? if its dealing with a specific problem you can leave a comment here describing why.

Reply:
I forgot to go back to review this bit. I wanted to match Ambassador with Masonry's styling on members and contributors block. So I hunted those down in DOM and copied and pasted them.